ArchUnitNETTests/Fluent/Syntax/Elements/ObjectsShouldTests.cs

```diff
@@ -0,0 +1,3229 @@
+            .Should()
+            .NotHaveAnyAttributes(helper.Attribute2)
+            .AssertNoViolations(helper);
+        Types()
+            .That()
+            .Are(helper.ClassWithoutAttributes, helper.OtherClassWithAttributes)
+            .Should()
+            .NotHaveAnyAttributes(helper.Attribute1)
+            .AssertAnyViolations(helper);
+        await helper.AssertSnapshotMatches();
+    }
+
+    [Fact]
+    public async Task NotHaveAnyAttributesThatTest()
+    {
+        var helper = new AttributeAssemblyTestHelpers();
+        helper.AddSnapshotHeader("No violations");
+        var should = Types().That().Are(helper.ClassWithSingleAttribute).Should();
+        should.NotHaveAnyAttributesThat().Are(helper.UnusedAttribute).AssertNoViolations(helper);
+
+        helper.AddSnapshotHeader("Violations");
+        should = Types().That().Are(helper.ClassWithSingleAttribute).Should();
+        should.NotHaveAnyAttributesThat().Are(helper.Attribute1).AssertOnlyViolations(helper);
+        await helper.AssertSnapshotMatches();
+    }
+
+    [Fact]
+    public async Task NotHaveAnyAttributesWithArgumentsTest()
+    {
+        var helper = new AttributeAssemblyTestHelpers();
+        helper.AddSnapshotHeader("No violations with type arguments");
+        var should = Types().That().Are(helper.ClassWithArguments).Should();
+        should
+            .NotHaveAnyAttributesWithArguments(helper.UnusedTypeParameterValue)
+            .AssertNoViolations(helper);
+        should
+            .NotHaveAnyAttributesWithArguments([helper.UnusedTypeParameterValue])
+            .AssertNoViolations(helper);
+
+        helper.AddSnapshotHeader("No violations with value arguments");
+        should = Types().That().Are(helper.ClassWithArguments).Should();
+        should
+            .NotHaveAnyAttributesWithArguments(helper.UnusedParameterValue)
+            .AssertNoViolations(helper);
+        should
+            .NotHaveAnyAttributesWithArguments([helper.UnusedParameterValue])
+            .AssertNoViolations(helper);
+
+        helper.AddSnapshotHeader("Violations with type arguments");
+        should = Types().That().Are(helper.ClassWithArguments).Should();
+        should
+            .NotHaveAnyAttributesWithArguments(helper.Attribute1Parameter3Value)
+            .AssertOnlyViolations(helper);
+        should
+            .NotHaveAnyAttributesWithArguments([helper.Attribute1Parameter3Value])
+            .AssertOnlyViolations(helper);
+
+        helper.AddSnapshotHeader("Violations with value arguments");
+        should = Types().That().Are(helper.ClassWithArguments).Should();
+        should
+            .NotHaveAnyAttributesWithArguments(helper.Attribute1Parameter2Value)
+            .AssertOnlyViolations(helper);
+        should
+            .NotHaveAnyAttributesWithArguments([helper.Attribute1Parameter2Value])
+            .AssertOnlyViolations(helper);
+
+        helper.AddSnapshotHeader("Type without attrubites");
```

Comment:
```suggestion
        helper.AddSnapshotHeader("Type without attributes");
```
2,000 lines into the file and this is the first typo I could find. 😄